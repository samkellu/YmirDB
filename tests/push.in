set a 123 33 554
set b 11 a 46 22
get a
push b
set c 123 334 b
push a 44 22 c
get a
bye
