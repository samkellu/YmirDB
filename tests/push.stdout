==846== Memcheck, a memory error detector
==846== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==846== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==846== Command: ./ymirdb
==846== 
==846== error calling PR_SET_PTRACER, vgdb might block
> ok

> ok

> [123 33 554]

> ok

> ok

> ok

> [c 22 44 123 33 554]

> bye
==846== 
==846== HEAP SUMMARY:
==846==     in use at exit: 0 bytes in 0 blocks
==846==   total heap usage: 53 allocs, 53 frees, 3,224 bytes allocated
==846== 
==846== All heap blocks were freed -- no leaks are possible
==846== 
==846== For lists of detected and suppressed errors, rerun with: -s
==846== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
